scss/base/_layout.scss

```diff
@@ -5,7 +5,7 @@
   display: none;
 
   @include respond-to('medium') {
-    @include absolute(top 10px left 10px);
+    @include absolute(top 8px left 20px);
```

Comment:
Just pushed it on `develop`: https://github.com/SassDoc/sassdoc-theme-default/commit/e25bca2250a0e637d23711d2f1db4bc6cf452430.